src/views/forms/AccountManagementForms/WithdrawForm/WithdrawButtonAndReceipt.tsx

```diff
@@ -63,38 +60,35 @@ export const WithdrawButtonAndReceipt = ({
   const { usdcLabel } = useTokenConfigs();
   const { connectionError } = useApiState();
 
-  const feeSubitems: DetailsItem[] = [];
-
-  if (typeof summary?.gasFee === 'number') {
-    feeSubitems.push({
-      key: 'gas-fees',
-      label: <span>{stringGetter({ key: STRING_KEYS.GAS_FEE })}</span>,
-      value: <Output type={OutputType.Fiat} value={summary?.gasFee} />,
-    });
-  }
-
-  if (typeof summary?.bridgeFee === 'number') {
-    feeSubitems.push({
-      key: 'bridge-fees',
-      label: <span>{stringGetter({ key: STRING_KEYS.BRIDGE_FEE })}</span>,
-      value: <Output type={OutputType.Fiat} value={summary?.bridgeFee} />,
-    });
-  }
-
-  const hasSubitems = feeSubitems.length > 0;
-
-  const showSubitemsToggle = showFeeBreakdown
-    ? stringGetter({ key: STRING_KEYS.HIDE_ALL_DETAILS })
-    : stringGetter({ key: STRING_KEYS.SHOW_ALL_DETAILS });
-
-  const totalFees = (summary?.bridgeFee || 0) + (summary?.gasFee || 0);
-
   const submitButtonReceipt = [
     {
-      key: 'total-fees',
-      label: <span>{stringGetter({ key: STRING_KEYS.TOTAL_FEES })}</span>,
-      value: <Output type={OutputType.Fiat} value={totalFees} />,
-      subitems: feeSubitems,
+      key: 'expected-amount-received',
+      label: (
+        <span>
+          {stringGetter({ key: STRING_KEYS.EXPECTED_AMOUNT_RECEIVED })}{' '}
+          {withdrawToken && <Tag>{withdrawToken?.symbol}</Tag>}
+        </span>
```

Comment:
instead of adding `{' '}`, can we style the `span` with `layoutMixins.row` so there's a gap instead?

Reply:
100% - for single style items do you mind using inline styles? or do we prefer to always create a styled component?

Reply:
nevermind, i'm going to make a component. i see this same issue pops up in `minimum amount received`